.github/CODEOWNERS

```
@@ -703,3 +703,11 @@ sdk/trafficmanager/Microsoft.Azure.Management.TrafficManager/            @tmsupp
 /eng/pipelines/aggregate-reports.yml            @AlexGhiondea @maririos
 /sdk/eventhub/tests.data.yml  @serkantkaraca @sjkwak
 /sdk/servicebus/tests.data.yml @shankarsama @DorothySun216 
+
+# ######## DPG ########
+
+# Onboarding Documentation and Quickstarts
+/doc/Data\%20Plane\%20Code\%20Generation           @chunyu3 @pshao25 @fengzhou-msft @lirenhe @annelo-msft
```

```suggestion
/doc/Data\ Plane\ Code\ Generation           @chunyu3 @pshao25 @fengzhou-msft @lirenhe @annelo-msft
```

If you decide against removing the spaces, I believe this is the format that you'd need.